guides/source/7_1_release_notes.md

```diff
@@ -23,7 +23,39 @@ Major Features
 
 ### Add default Dockerfiles for Rails applications
```

Comment:
Do we need to link the title to PR?

Reply:
Yeah, true. I'd like to link it somewhere in the text, but since some titles will not have where to link to, it is better to not add the link in the title.

Reply:
I've already added that in the description